server/db_mysql.go

```diff
@@ -195,6 +196,39 @@ func (ms *MsqlCache) GetFixity(start string, end string, item string, status str
 	return results
 }
 
+func (ms *MsqlCache) PostFixity(item string) (int, error)  {
+        var err error
+        fixity_results := ms.GetFixity("*", "*", item, "scheduled")
+        if len(fixity_results) > 0 {
```

Comment:
Why is the code deleting up to one scheduled fixity check for the item?

Don't delete any other scheduled checks.

Reply:
It has to. UpdateFixity adds a check for now if the item doesn't exist; otherwise, in our case, it would update all  od the  next scheduled check fields except the one we want, scheduled_time.